Podfile

```diff
@@ -4,4 +4,5 @@ target 'TreeTracker' do
   use_frameworks!
   pod 'SwiftLint'
   pod 'SwiftGen', '~> 6.0'
+  pod 'SDWebImage', '~> 5.0'
```

Comment:
I can't see this used anywhere? We should probably discuss if we want to include this dependency or not. My personal preference would be not to.

Reply:
I used it earlier, but later changed. Will remove it